docs/specs/test-specs.md

```diff
@@ -1,39 +1,114 @@
-# Dusty Testing Specs
+# Test Specs
+
 These specs are used to specify tests you can run in apps and libs. These do not exist as distinct documents but as subfields of lib and app specs. They are found under the **test** key.
 
-## Keys
+## image
+
+```
+image: ubuntu:15.04
+```
+
+`image` specifies a Docker image on which to base the container which will be used
+to run the test suite. If the image does not exist locally, Dusty will pull it.
```

Comment:
this is not true.  Image is used to build a second image which will be used to build the container.